django/contrib/auth/admin.py

```diff
@@ -176,7 +178,8 @@ def user_change_password(self, request, id, form_url=''):
 
         request.current_app = self.admin_site.name
 
-        return TemplateResponse(request,
+        return TemplateResponse(
+            request,
```

Comment:
well, I'd suggest to revert this change entirely as it's unrelated to the patch.